xml/containers-basics.xml

```diff
@@ -12,12 +12,17 @@
   </dm:docmanager>
  </info>
  <para>
-  Linux containers offer a lightweight virtualization method to run multiple
-  virtual environments (containers) simultaneously on a single host. Unlike
+  The Linux kernel has some powerful capabilities to put system processes in isolated environments
+  called containers. Two specific Linux Kernel features, namespaces and kernel control groups
+  (see <xref linkend="cha-tuning-cgroups"/>) implements the container isolation from the rest of 
+  the host system or other containers. 
+  Thus Linux containers offer a lightweight "virtualization method" to run multiple
+  isolated environments (containers) simultaneously on a single host. Unlike
   technologies like &xen; or &kvm;, where the processor simulates a complete
-  hardware environment and a hypervisor controls virtual machines, containers
-  provide virtualization at the operating system level, where the kernel
-  controls the isolated containers.
+  hardware environment through a virtualization layer, Linux containers share and use directly the
+  host OS kernel.
+  This increase the overall performance as there is no hardware emulation, thus minimizing the
+  kernel overhead required for accessing the operating system ressources.
```

Comment:
This sentence is strictly not correct as modern CPUs have virtualization support and can run VMs at nearly native speeds.
```suggestion
```

Reply:
Ok, I know this sound like marketing material and not 100% technically true. I heard that there is some Micro-VM or else that are super fast/efficient... Still aren't container more efficient that our own XEN/KVM images?

Reply:
They can be, but they don't have to. As usual, the answer is: it depends.

A VM that can passthrough nearly all its heavy lifting directly to the hardware will run at native speed (e.g. a HPC application). A container that performs a ton of syscalls on the other hand will suffer a noticeable performance penalty, as the kernel will have to check each syscall, whether the container is allowed to perform it.

Reply:
then how about turning this around a bit: 

> This decrease the kernel overhead required for accessing the operating system resources and in turn allow for better performance in general, compare to a generic Virtual Machine environment.

I really care about this statement and I think it should be fine if we are not 100% technically correct.